gradle.properties

```diff
@@ -7,6 +7,6 @@ org.gradle.jvmargs=-Xmx1G
 	plasma_build=18
 
 # Mod Properties
-	mod_version = 1.1.0
-	maven_group = io.github.minecraftcursedlegacy
+	mod_version = 1.1.1
```

Comment:
is this worth bumping ver for?

Reply:
Beats me. I changed it because I modified the buildscript and made some minor changes to fix warnings (but I didn't push them because they were *so* minor.)